ydb/core/grpc_services/db_metadata_cache.h

```diff
@@ -44,6 +45,9 @@ class TDatabaseMetadataCache : public TActorBootstrapped<TDatabaseMetadataCache>
     TActorId PublishActor;
     TActorId SubscribeActor;
     bool RequestInProgress = false;
+    ::NMonitoring::TDynamicCounterPtr Counters;
+    static const inline TString HEALTHCHECK_REQUESTS_MADE_COUNTER = "DbMetadataCache/HealthCheckRequestsMade";
```

Comment:
I think people are more used to terms like "hit" or "miss"

Reply:
True, but RequestsMade aren't cache misses in the traditional sense - they are made periodically by the cache itself. You don't think calling them cache misses would be misleading?

Reply:
it would. okay then. maybe we should add some sensors on grpc side. is there any flag or status in response data which could tell us if the response was generated from a cache?